startup.py

```diff
@@ -97,7 +97,16 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
             "basic",
             "bootstrap.py"
         )
-
+        #To be removed when Python Package is integrated properly
+        #Code is passing a PySide2 environment variable to Shotgun unity to use for install
+        #A proper integration would simple Pip install Pyside
+        from sgtk.util.qt_importer import QtImporter
+        r = QtImporter()._import_pyside2()
+        PySideInstance = r[2].__path__[0]
```

Comment:
Normally a variable should start with a lower-case. Also it's a filename not an instance of the module.